vuepress/docs/next/docs/ecr/ecr-how-setup-private-git-repo.md

```diff
@@ -0,0 +1,106 @@
+# ECR: How to checkout Bundles from a private Git repository
```

Comment:
1) Maybe rename this file to either ecr-setup-private-git-repo.md or ecr-how-to-setup-private-git-repo.md since it's visible in the URL. 
2) I also sent a PR with a few language changes to this file: https://github.com/entando/entando-docs/pull/205
3) This should probably be added to the navigation under .vuepress/next.js.